src/java.base/share/classes/sun/security/pkcs12/PKCS12KeyStore.java

```diff
@@ -2316,7 +2306,6 @@ private X509Certificate findIssuer(X509Certificate input) {
                             return cert;
                         } else {
                             // Different keyId. Not a fallback.
-                            continue;
```

Comment:
Personally I prefer seeing "continue" here - it makes it clear that none of the remaining code in this loop is relevant.

Reply:
Agreed. Adding it back.